From 7c2f3e7348826a008a6c6e130ad21d5479b77f69 Mon Sep 17 00:00:00 2001
From: "Luke D. Jones" <luke@ljones.dev>
Date: Thu, 11 Aug 2022 13:22:09 +1200
Subject: [PATCH 03/17] Fixes: ca91ea34778f ("asus-wmi: Add panel overdrive
 functionality")

The panel_od attribute was not documented, this adds the
required documentation.

Signed-off-by: Luke D. Jones <luke@ljones.dev>
---
 Documentation/ABI/testing/sysfs-platform-asus-wmi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Documentation/ABI/testing/sysfs-platform-asus-wmi b/Documentation/ABI/testing/sysfs-platform-asus-wmi
index b984bdcd0c40..574b5170a37d 100644
--- a/Documentation/ABI/testing/sysfs-platform-asus-wmi
+++ b/Documentation/ABI/testing/sysfs-platform-asus-wmi
@@ -74,5 +74,14 @@ Contact:       "Luke Jones" <luke@ljones.dev>
 Description:
                Enable the external GPU paired with ROG X-Flow laptops.
                Toggling this setting will also trigger ACPI to disable the dGPU:
+                       * 0 - Disable,
+                       * 1 - Enable,
+
+What:          /sys/devices/platform/<platform>/panel_od
+Date:          Aug 2022
+KernelVersion: 5.17
+Contact:       "Luke Jones" <luke@ljones.dev>
+Description:
+               Enable an LCD response-time boost to reduce or remove ghosting:
                        * 0 - Disable,
                        * 1 - Enable,
\ No newline at end of file
-- 
2.37.1

